### ISSUE TITLE
Top-level exception handler that asks the user to file a bug report

### ISSUE THREAD

**Issue by lexelby** [inkstitch/inkstitch#185]
Users should never see a Python exception traceback -- even if they did something "wrong".  Ink/Stitch should catch erroneous conditions and offer the user helpful advice on how to correct the situation.

Add a top-level exception handler that asks the user to file an issue here.

Consider integrating an exception tracker service if there exists one that offers a free plan for open source projects.

**Comment by atrus6** [comment]
Related to seeing python errors, should we report when Ink/Stitch throws a python exception?

**Comment by lexelby** [comment]
Most definitely.  Any time you see an exception, that's a bug that I want to know about.

**Comment by kaalleen** [comment]
Did we lose this bit of helpful information when removing stub.py?

**Comment by lexelby** [comment]
Nope!  `inkstitch.py` is still there, and it contains the exception handler.

**Comment by kaalleen** [comment]
Ok, good. Grep couldn't find parts of the info-text ([...]If you're having trouble, please file an issue here, including the text below: https://github.com/inkstitch/inkstitch/issues [...]) in the source anymore . So I was worried it went away ;)

**Comment by lexelby** [comment]
Oh, I was confused.  You're right, that text is no longer there because it was in the stub only.  And yes, you were right, we did lose that when I got rid of the stub.  But really, the "If you're having trouble" was only for problems when the stub was trying to launch Ink/Stitch.

We do have a top-level exception handler in inkstitch.py, but it just prints the exception traceback and exits.  We haven't yet expanded it into something like what I described above.
